src/css/modules/_SectionTitle.scss

```diff
@@ -0,0 +1,16 @@
+.coa_SectionTitle {
+  margin: $coa-space-level7 auto $coa-space-level6;
+  text-transform: uppercase;
+  text-align: center;
+  font-size: $h2-font-size;
+  font-weight: bold;
+  max-width: 100% !important; // needed to override coa0-main textwrap. TODO: talk to simi
```

Comment:
Oh, this is similar to the comment below. @cthibaultatx , initially had limitations on text width to keep it readable on large screens. I'm not sure if she intended to stick with the limitation, but with text centered instead. Just a place to get clarification. Also, like below, we may wanna toss existing css so we don't fight with it. Seems like things have changed enough to do so.